nixos/modules/services/aira/github-ipfs.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.github-ipfs;
+
+in {
+  options = {
+    services.github-ipfs = {
+      enable = mkEnableOption "Enable github-ipfs service."; 
+
+      key = mkOption {
+        type = types.str;
+        default = "none";
+        description = "GitHub Webhook secret";
+      };
+
+      path = mkOption {
+        type = types.str;
+        default = "/registry";
+        description = "Absolute registry path in container";
+      };
+
+      port = mkOption {
+        type = types.int;
+        default = 8000;
```

Comment:
Add port to firewall rules like https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/networking/ssh/sshd.nix#L313